http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/AzureSubscriber.scala

```diff
@@ -0,0 +1,13 @@
+package org.broadinstitute.dsde.workbench.leonardo.dao
+
+import fs2.Stream
+import org.broadinstitute.dsde.workbench.model.TraceId
+
+import java.time.Instant
+
+//TODO: Jesus will fill this out
+trait AzureSubscriber[F[_], A] {
```

Comment:
Probably needs to be converted to Scala, but example of using Azure Service Bus from Java: https://github.com/DataBiosphere/stairway/blob/develop/stairway-azure/src/main/java/bio/terra/stairway/azure/AzureServiceBusQueue.java